openspec/changes/enterprise-01-module-policy-client/specs/enterprise-policy-client/spec.md

```
@@ -0,0 +1,28 @@
+## ADDED Requirements
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)